app/ethWallet-geth.js

```diff
@@ -104,29 +106,34 @@ const spawnGeth = async () => {
   }
 
   const gethOptions = {
-    stdio: process.env.GETH_LOG ? 'inherit' : 'ignore'
+    stdio: ((envNet === 'ropsten') || process.env.GETH_LOG) ? 'inherit' : 'ignore'
```

Comment:
is this included intentionally? i think that would should let `GETH_LOG` be the sole deciding indicator for logging, lest there is value in always having it debug on `ropsten`

Reply:
yes. it's intentional: if someone is on the test network, or someone turns on GETH_LOG, then we get logs, otherwise not. in other words, for production use, no logs by default; for testnet use, logs by default.

Reply:
++